src/app.rb

```diff
@@ -86,6 +103,7 @@ def create_rental
     needed_book.add_rental(date, needed_person)
     rental = Rental.new(date, needed_book, needed_person)
     $rental_list << rental
+    $storage.add_rental rental
```

Comment:
- [ ] Creating a rental also throws an error. The error seems to come from **`line: 88`** which is part of the `create_rental` method. Line `106` is the only new change in the `create_rental` method, so that's why you see the marker at this point. But please check the **`line 88`**

Screenshot |
:-------------:|
<img width="675" alt="Screen Shot 2022-11-08 at 11 03 31 PM" src="https://user-images.githubusercontent.com/39523312/200695548-01e770bb-73e6-4152-a98e-6704fcb4f572.png"> |